tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -616,6 +616,11 @@ public static partial class Tags
         /// </summary>
         internal const string IastJsonTagSizeExceeded = "_dd.iast.json.tag.size.exceeded";
 
+        /// <summary>
+        /// Indicates if the vulnerability meta struct has been truncated because it exceeds the maximum tag size
+        /// </summary>
+        internal const string IastMetaStructTagSizeExceeded = "_dd.iast.metastruct.tag.size.exceeded";
```

Comment:
Should we start nesting the IAST tags like we do for AppSec and others? Some are nested at the bottom of these file, and AppSec tags are in `tracer/src/Datadog.Trace/Tags.AppSec.cs`.

Reply:
I won't do that in that PR, but yea that might be a good idea to nest them like the others 😄